project1/server/server_tasks/tport_task.h

```diff
@@ -0,0 +1,27 @@
+/**
+ * @File Task for monitoring port for 'Terminate' Commands
+ */
+
+#ifndef PROJECT1_TPORT_TASK_H
+#define PROJECT1_TPORT_TASK_H
+
+#include "server_task.h"
+#include "../../thread_pool/task.h"
+#include "../../thread_pool/thread_pool.h"
+#include "command_ids.h"
+
+namespace server_tasks {
+
+    class TPortTask : public ServerTask {
```

Comment:
(minor) I know the professor calls them `nport` and `tport`, but in general, these are stupid names. Since we can name things however we want in our code, I would call these `NormalPortTask` and `TerminatePortTask`, or something along those lines.

Reply:
I agree. I've thought about this, and figured I'd change them after funcionality has been verified.